src/utils.js

```diff
@@ -8,3 +8,31 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findAverage = (...numbers) => {
+  let ave = 0;
+  let res = numbers.every((element) => {return Number.isInteger(element);});
+  
+  try {
+    if(numbers.length < 1) {
+      throw "This function expects at least one argument";
```

Comment:
Change all occurences of throw to throw new Error("error message")

Reply:
ok